tests/asset.test.ts

```diff
@@ -0,0 +1,92 @@
+import { Livepeer } from "../index.js";
+import { compareResponseStructures } from "./utils";
+
+const API_BASE_URL = "https://livepeer.studio/api";
+const apiKey = "478de893-583e-47a6-9294-da65a98c6234";
```

Comment:
I started a PR and will open it after we merge this, but we should remove these API keys from being hardcoded in the repo and then delete them in your dashboard, so someone doesn't hijack your account.